AlertaDengue/dados/dbdata.py

```diff
@@ -950,6 +952,59 @@ def load_cases(
             return pd.DataFrame(result.fetchall(), columns=result.keys())
 
 
+class AlertaState:
```

Comment:
This class is named AlertaState, but it does not keep state, i.e., there are no instance level variables declared. nothing wrong, just a comment.